common-lib/src/main/scala/com/gu/mediaservice/lib/BaseStore.scala

```diff
@@ -25,19 +22,7 @@ abstract class BaseStore[TStoreKey, TStoreVal](bucket: String, config: CommonCon
   protected val store: Box[Map[TStoreKey, TStoreVal]] = Box(Map.empty)
   protected val lastUpdated: Box[DateTime] = Box(DateTime.now())
 
-  protected def getS3Object(key: String): Option[String] = {
```

Comment:
This moves to the more generic `S3` class - https://github.com/guardian/grid/pull/2767/files#diff-9e12ba48c7d2af2248094256c01c3d36R100